cmd/kubeadm/app/util/system/docker_validator_test.go

```diff
@@ -28,7 +28,7 @@ func TestValidateDockerInfo(t *testing.T) {
 		Reporter: DefaultReporter,
 	}
 	spec := &DockerSpec{
-		Version:     []string{`1\.1[1-3]\..*`, `17\.03\..*`}, // Requires [1.11, 17.03].
+		Version:     []string{`1\.1[1-3]\..*`, `18\.06\..*`}, // Requires [1.11, 18.06].
```

Comment:
@yuansisi 
sorry but we've missed adding some changes to the tests.

make this test have `warn: false` instead of `warn: true`:
```
		{
			info: types.Info{Driver: "driver_2", ServerVersion: "17.06.0-ce"},
			err:  false,
			warn: true,
		},
```

add this new test in there too:
```
		{
			info: types.Info{Driver: "driver_2", ServerVersion: "18.06.0-ce"},
			err:  false,
			warn: false,
		},
```

Reply:
@yuansisi - ping.  We're getting low on time now.